uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AadLoginComponentHandler.java

```diff
@@ -97,7 +100,7 @@ public void handleAccountPicker(@NonNull final String username) {
     }
 
     private UiObject getConsentScreen() {
-        return UiAutomatorUtils.obtainUiObjectWithResourceId("consentHeader");
+        return UiAutomatorUtils.obtainUiObjectWithResourceId("appDomainLinkToAppInfo");
```

Comment:
This is a strange change, but if it works then we can take it.

Reply:
I'm thinking of proposing a change to how we handle resource-ids, basically create a way to fall back on multiple resource ids incase the first doesn't work. don't know when I'll get to that but I need to ideate a little more on it.